src/alivedetection.c

```diff
@@ -1295,6 +1296,7 @@ send_dead_hosts_to_ospd_openvas (void)
         {
           g_string_append (chunked_hosts, host_str);
           hosts_in_chunk++;
+          count_dead_ips++;
```

Comment:
I think its good enough for now but keep in mind that we might have edge cases here when `max_alive_hosts` is used because of the way the progress bar and resume scan works. Alive hosts might be counted as dead hosts because they were not allowed to be considered as alive.

Reply:
OK. But nothing to address in this particular PR I guess.